USFMConverter/UI/Pages/OptionView.axaml.cs

```diff
@@ -23,39 +28,84 @@ private void InitializeComponent()
             blurredArea.AddHandler(PointerPressedEvent, OnCloseClick);
 
             outputFormatCb = this.FindControl<ComboBox>("OutputFormatSelector");
-            outputFormatCb.AddHandler(ComboBox.SelectionChangedEvent, OnOuputFormatSelect);
+            int lastUsedFormatIndex = GetLastUsedFormatIndex(outputFormatCb.SelectedIndex);
+            outputFormatCb.SelectedIndex = lastUsedFormatIndex;
+            ShowFormatView();
 
+            outputFormatCb.AddHandler(ComboBox.SelectionChangedEvent, OnOuputFormatSelect);
+            
             optionView = this.FindControl<UserControl>("OptionView");
         }
 
         private void OnOuputFormatSelect(object? sender, SelectionChangedEventArgs e)
         {
-            string selectedFormat = ((ComboBoxItem)outputFormatCb.SelectedItem).Tag.ToString();
-            
+            string selectedFormat = ((ComboBoxItem) outputFormatCb.SelectedItem).Tag.ToString();
+            ShowFormatView();
+            SaveOptions();
+            LoadOptions(selectedFormat);
+        }
+
+        private void ShowFormatView()
+        {
+            string selectedFormat = ((ComboBoxItem) outputFormatCb.SelectedItem).Tag.ToString();
+
             foreach (ComboBoxItem comboBoxItem in outputFormatCb.Items)
             {
                 var formatName = comboBoxItem.Tag.ToString();
                 this.FindControl<UserControl>(formatName).IsVisible = (formatName == selectedFormat);
             }
-
-            ResetInputOptions();
         }
+        
 
         private void OnCloseClick(object? sender, RoutedEventArgs e)
         {
             optionView.IsVisible = false;
+            SaveOptions();
+        }
+
+        private void SaveOptions()
+        {
+            // Save config files when option view is closed
+            var dataContext = (ViewData) DataContext;
+            FileSystem.SaveOptionConfig(dataContext);
+            FileSystem.SaveLastUsedFormat((ViewData) DataContext);
```

Comment:
Can these two static functions merged into one? They're doing pretty much similar thing: write the user settings/config to the setting files